.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,16 +9,15 @@
 
 <!--
 - Make a GitHub Access token here https://github.com/settings/tokens
-- `cp secrets.txt.example secrets.txt`
-- Update the value of GITHUB_TOKEN in secrets.txt
+- export GITHUB_TOKEN as an environment variable with the token above
```

Comment:
maybe add something about storing the token, since you can only read it once on github

Reply:
I have updated with more information, let me know what you think.